cms/envs/common.py

```diff
@@ -89,7 +89,9 @@
     FILE_UPLOAD_STORAGE_BUCKET_NAME,
     FILE_UPLOAD_STORAGE_PREFIX,
 
-    COURSE_ENROLLMENT_MODES
+    COURSE_ENROLLMENT_MODES,
+
+    SITE_SUPPORT_LINK
```

Comment:
You have this in your [PR](https://github.com/edx/edx-platform/pull/15017/files#diff-a8eedcb5656641698b92541a82609962R94) for ENT-328 as `SUPPORT_SITE_LINK` which I think is correct. Maybe remove this change from this PR?